packages/augur-ui/src/modules/events/actions/add-update-transaction.ts

```diff
@@ -30,6 +45,16 @@ export const addUpdateTransaction = (txStatus: Events.TXStatus) => (
         }
         break;
       }
+      case CREATEMARKET:
+      case CREATECATEGORICALMARKET:
+      case CREATESCALARMARKET:
+      case CREATEYESNOMARKET: {
+        if (hash) dispatch(addPendingData(hash, CREATE_MARKET, eventName, transaction.params));
```

Comment:
build `info` typed object here. add another property `txParams` for the transaction params (which will be used for retry)